packages/flutter/test/gestures/force_press_test.dart

```diff
@@ -109,6 +109,99 @@ void main() {
     expect(ended, 1);
   });
 
+  testGesture('Force presses are not recognized on devices with low maxmium pressure', (GestureTester tester) {
+    // Device specific constants that represent those from the iPhone X
```

Comment:
This comment isn't right right?

Reply:
I may have been referring to the .4 and the .85 being the start and peak pressure values as perceived on the iPhone X relative to a pressure max of 1.0. But yah, that is not the clearest comment given there is interpolation happening behind the scenes.